index.html

```diff
@@ -14,18 +14,17 @@
           <span class="p-hamburger__line"></span>
         </button>
         <nav class="p-global-nav" aria-hidden="true" id="js-global-nav">
-          <ul　 class="p-global-nav__list">
+          <ul class="p-global-nav__list">
             <li class="p-global-nav__item"><a href="http://">メニュー1</a></li>
             <li class="p-global-nav__item"><a href="http://">メニュー2</a></li>
             <li class="p-global-nav__item"><a href="http://">メニュー3</a></li>
             <li class="p-global-nav__item"><a href="http://">メニュー4</a></li>
             <li class="p-global-nav__item"><a href="http://">メニュー5</a></li>
-          </ul><!-- /.p-global-nav__list-->
+          </ul>
         </nav><!-- /.p-header-nav -->
       </div><!-- /.l-inner -->
     </header><!-- /.l-header -->
```

Comment:
please remove those as same.
```js
   </nav><!-- /.p-header-nav -->
      </div><!-- /.l-inner -->
    </header><!-- /.l-header -->
    ```

Reply:
Thank you.
Let me ask you a question.
Is it because this comment out is meaningless?

Reply:
yes. [see](https://github.com/taka0209/js_lesson/pull/10#discussion_r659065149)